lib/spec/staticcacher.sw-spec.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+importScripts('/base/lib/staticcacher.js');
+
+describe('Static cacher', function() {
+
+  var defaultCacheName = 'offline';
+  var defaultUrls = ['http://www.example.com/', 'http://www.mozilla.org/'];
+  var defaultStaticCacher = new StaticCacher(defaultUrls);
+
+  sinon.stub(self, 'fetch', function() {
+    var response = new Response('content', {});
```

Comment:
I recommend to add some kind of bound between the request and the response. For instance, suppose this map:

```js
var requestMap = {
  'http://www.example.com/': 'response1',
  'http://www.mozilla.org/': 'response2',
};
var defaultUrls = Object.keys(requestMap);

sinon.stub(self, 'fetch', function (req) {
  var response = new Response(requestMap[req.url], {});
  return Promise.resolve(response);
});
```
Now, when asserting on the records inside the `'offline'` cache you can check if the proper responses where added to their respective requests.